data/map.txt

```diff
@@ -26321,8 +26321,9 @@ planet "Rekat Moraski"
 planet Relic
 	attributes "near earth"
 	landscape land/valley4
-	description `As a borderline-habitable planet in one of the systems neighboring Sol, Relic was colonized hundreds of years ago in the early years of the space race. Because of its low gravity and limited natural resources, only a few minor settlements are left.`
-	spaceport `The spaceport on Relic consists of one low building surrounded by open-air concrete slabs for ships to land on. If rain were more common here, you would worry about your ship rusting, but in this environment the only threat to it is the infrequent sand storms.`
+		description `Relic was colonized over seven hundred years ago, back when the construction of the first hyperdrive was still in living memory. At this point, most starships had a very limited range, so the stars adjacent to earth where the only viable opinion for any colonization attempts. Because of this, Relic seemed like it was going to become a major colony, despite it's low gravity, harsh environment, and limited resources.`
+	description `However, this would not last. In less than fifty years, the colonies had spread to much more favourable planets, and Relic was left behind by most human development.`
+	spaceport `The spaceport here is located in the only recognized city on Relic, and it's mostly designed to handle the occasional merchant or navy ship. Almost all the ships that do arrive are here to refuel, since the industry here is made up of a few factories and a trading post. This leads to a running joke among the locals that the chief industry here is the gas station.`
```

Comment:
I'd keep the old description's characterization of the spaceport as being just a bunch of concrete slabs for ships to land on instead of having proper landing pads. It helps to better show just how poor and crappy this spaceport is.
Also, Navy.
```suggestion
	spaceport `The spaceport here is located in the only recognized city on Relic, and it's mostly designed to handle the occasional merchant or Navy ship. Almost all the ships that do arrive are here to refuel, since the industry here is made up of a few factories and a trading post. This leads to a running joke among the locals that the chief industry here is the gas station.`
```